src/Sarif.Sarifer/FileAndForgetEventName.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Sarif.Sarifer
+{
+    internal static class FileAndForgetEventName
```

Comment:
>FileAndForgetEventName [](start = 26, length = 22)

What does 'file' mean in FileAndForget? Was this supposed to be 'fire'?

Reply:
Heaven knows I agree with you, and I've typoed it that way myself. But `FileAndForget` is defined in `Microsoft.VisualStudio.Shell`, an extension method on `System.Threading.Tasks.Task`. I supposed it means "file it away in a drawer and forget about it".

---
In reply to: [520855003](https://github.com/microsoft/sarif-visualstudio-extension/pull/265#discussion_r520855003) [](ancestors = 520855003)

Reply:
Equally likely it is simply a mistake that everyone continues to promulgate.

---
In reply to: [520859565](https://github.com/microsoft/sarif-visualstudio-extension/pull/265#discussion_r520859565) [](ancestors = 520859565,520855003)

Reply:
Not sure what you mean by that. I agree it was a mistake when it was named, but now it's part of the VS SDK's public API.

---
In reply to: [520862198](https://github.com/microsoft/sarif-visualstudio-extension/pull/265#discussion_r520862198) [](ancestors = 520862198,520859565,520855003)

Reply:
Oh, you mean "why did I name _my_ class" with the same odd name as the SDK's class. To make clear it's connection to the SDK's class.

---
In reply to: [520864070](https://github.com/microsoft/sarif-visualstudio-extension/pull/265#discussion_r520864070) [](ancestors = 520864070,520862198,520859565,520855003)